test/jdk/java/lang/Thread/virtual/stress/Skynet.java

```diff
@@ -35,7 +39,7 @@
  * @requires vm.gc.Z
  * @compile --enable-preview -source ${jdk.version} Skynet.java
  * @run main/othervm/timeout=300 --enable-preview -XX:+UnlockDiagnosticVMOptions
- *     -XX:+ZVerifyViews -XX:ZCollectionInterval=0.01 Skynet
+ *     -XX:+ZVerifyViews -XX:ZCollectionInterval=0.01 -Xmx1g Skynet
```

Comment:
OpenJ9 doesn't include vm.gc.Z so this modification isn't required.

Reply:
fixed